src/plugins/event_annotation/server/fetch_event_annotations/index.ts

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+// import { StartServicesAccessor } from '@kbn/core/server';
```

Comment:
keeping this around for any reason?

Reply:
Yes - that's because we have to make fetching annotations work for the server too, not for Lens usage but for the future. I spent some time on it and couldn't make it work, and as it's not priority, I'd like to come back to it once all the basic pieces of Lens query annotations are ready. I'll add a todo comment, but won't remove it yet, as it's a good starting point for the work in the future.